source/dsm.js

```diff
@@ -1,5 +1,6 @@
 const camelCase = require('lodash.camelcase');
 const snakeCase = require('lodash.snakecase');
+const isError = require('lodash/isError');
```

Comment:
Currently, `lodash` is just a devDependency, not a production dependency.  I'd prefer if we found a different way of detecting Error.